### PR TITLE
[MM-96]: Added the test cases for the Settings slash command and zoom icon action in app bar on MM.

### PR DESCRIPTION
**Pull request by arush-vashishtha**
### Summary
This PR consists the test cases for the following scenarios,

- When the user uses the Zoom icon from the app bar to create a zoom meeting.
- When the user enter the slash command on MM.

### CODE REVIEW

File: data/test-cases/plugins/zoom/general/Settings_slash_commands.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Enter the slash command `/zoom` in the chat in any desired channel or DM/GM on MM.
```
Comment (AayushChaudhary0001) [inline data/test-cases/plugins/zoom/general/Settings_slash_commands.md]: Try to write enter and run when you do not have any other action in the same step.

**Review by AayushChaudhary0001 — CHANGES_REQUESTED** [review]

**Review by AayushChaudhary0001 — APPROVED** [review]

File: data/test-cases/plugins/zoom/general/Settings_slash_commands.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Enter and run the slash command `/zoom` in the chat in any desired channel or DM/GM on MM.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Settings_slash_commands.md]:
```suggestion
1. Enter and run the slash command `/zoom` in any desired channel or DM/GM on MM.
```

File: data/test-cases/plugins/zoom/general/Settings_slash_commands.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+# (Required) Ensure all values are filled up
+name: "Settings slash command should not entertain any command after `/zoom settings`"
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Settings_slash_commands.md]:
```suggestion
name: "Settings slash command should not accept any arguments after `/zoom settings`"
```

File: data/test-cases/plugins/zoom/general/Settings_slash_commands.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Enter and run the slash command `/zoom settings <command>` in the chat in any desired channel or DM/GM on MM.
+
+**Step 3**
+
+1. Enter and run the slash command `/zoom settings` in the chat in any desired channel or DM/GM on MM.
+
+**Expected**
+
+The suggested list of the commands should show the following suggestions in the desried channel or DM/GM on MM, `start[meeting topic]`, `connect`, `disconnect`, `settings` and `help`.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Settings_slash_commands.md]:
```suggestion
The suggested list of the commands should show the following suggestions in the desried channel or DM/GM on MM, `start [meeting topic]`, `connect`, `disconnect`, `settings` and `help`.
```

File: data/test-cases/plugins/zoom/general/Settings_slash_commands.md
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Settings_slash_commands.md:1]: File name should be `Settings_slash_command.md`

File: data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
+---
+# (Required) Ensure all values are filled up
+name: "Creating meeting through Zoom icon in the App bar/channel header."
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md]:
```suggestion
name: "Creating meeting using Zoom icon in the app bar/channel header."
```

File: data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Connect your MM account to your Zoom account using the slash command `/zoom connect`.
+2. Set the Zoom setting by running the slash command `/zoom settings` to `Yes` for using Personal Meeting ID when starting a meeting.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md]:
```suggestion
2. Set the Zoom setting by running the slash command `/zoom settings` to `Yes` for using Personal Meeting ID when creating a meeting.
```

File: data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+3. Click on the Zoom icon in the app bar/channel header when there is no other new Zoom meeting in the channel or DM/GM on MM.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md]:
```suggestion
3. Click on the Zoom icon in the app bar/channel header when there is no recent Zoom meeting in the channel or DM/GM on MM.
```
Check everywhere

File: data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md (hunk 6 above)
Comment (raghavaggarwal2308) [reply data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md]: Check everywhere

File: data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+**Step 5**
+
+1. Connect your MM account to your Zoom account using the slash command `/zoom connect`.
+2. Create a meeting by  clicking the Zoom icon in the app bar/channel header in any desired channel or DM/GM on MM.
+3. Click on the Zoom icon in the app bar/channel header in the desired channel or DM/GM on MM.
+
+**Expected**
+
+The user should be navigated to the Zoom meeting with the Personal Meeting ID on Zoom.
+After step 2, the user should be navigated to the Zoom meeting wtih Unique Meeting ID on Zoom.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md]:
```suggestion
After step 2, the user should be navigated to the Zoom meeting with Unique Meeting ID on Zoom.
```

File: data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+After step 3, the user should get the slack attachment to select the type of meeting to create in the desired channel or DM/GM on MM.
+After step 4, the user should get the message about already created Zoom meeting and should get the slack attachment with option to select `Create New Meeting` or `Join Existing Meeting` in the desired channel or DM/GM on MM.
```
Comment (raghavaggarwal2308) [inline data/test-cases/plugins/zoom/general/Zoom_icon_in_app_bar.md]:
```suggestion
After step 4, the user should get the message about an already created Zoom meeting and should get the slack attachment with the option to select `Create New Meeting` or `Join Existing Meeting` in the desired channel or DM/GM on MM.
```

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

**Review by raghavaggarwal2308 — APPROVED** [review]

**Pull request merged**